lib/cucumber-api/steps.rb

```diff
@@ -27,10 +27,39 @@
   end
 end
 
+Then (/^I grab "([^"]+)"$/) do |k|
```

Comment:
Also this would be a shortcut for the other grab step that infers grabbed ID from JSON path is it?